src/modules/SdnDiag.Health/private/Test-SlbManagerConnectionToMux.ps1

```diff
@@ -55,19 +55,19 @@ function Test-SlbManagerConnectionToMux {
             $virtualServerConnection = $virtualServer.properties.connections[0].managementAddresses
             $connectionExists = Invoke-PSRemoteCommand -ComputerName $virtualServerConnection -Credential $Credential -ScriptBlock $netConnectionExistsScriptBlock
             if (-NOT $connectionExists) {
-                "{0} is not connected to SlbManager of Network Controller" -f $virtualServerConnection | Trace-Output -Level:Exception
+                "{0} is not connected to SlbManager of Network Controller" -f $mux.resourceRef | Trace-Output -Level:Exception
```

Comment:
Can we include Mux ResourceRef and Virtual Server connection, as in some deployment the ResourceRef might just GUID

Reply:
The reason I went with resourceRef, is that you can then just call `Get-SdnResource -ResourceRef 'string'` to get the details.  I will update `Get-SdnLoadBalancerMux` and `Get-SdnGateway` to include the `-ResourceRef` and `-ResourceId` property that will make identify the FQDN or IP Address of the object.

Reply:
![image](https://github.com/microsoft/SdnDiagnostics/assets/18577812/dffc4c76-16ef-4f30-adb6-cf9652493222)

![image](https://github.com/microsoft/SdnDiagnostics/assets/18577812/4b3442e4-634b-4e1d-b038-267f48884e86)

Reply:
Is this viable?

Reply:
That's better now. Let's see if any user feedback for this after release.